swap_meet/clothing.py

```diff
@@ -1,2 +1,18 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+
+    def __init__(self, id=None, fabric="Unknown", condition=0):
+        
+        super().__init__(id, condition)
+        self.fabric = fabric
+        # self.fabric = fabric if not fabric is "Unknown" else fabric
```

Comment:
Great move to change how you assign self.fabric, just make sure to remove any commented code when submitting a PR!